### PR TITLE
Pdevi/yolov5 imgurl issue

### PR DESCRIPTION
**Pull request by pdevi5**

### CODE REVIEW

File: model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py [hunk 1]
```diff
@@ -65,11 +67,14 @@ def run_pytorch_yolov5_320(variant="yolov5s"):
     pixel_size = 320  # image pixel size
 
     # Load data sample
-    url = "https://raw.githubusercontent.com/pytorch/hub/master/images/dog.jpg"
-    image = Image.open(requests.get(url, stream=True).raw).convert("RGB")
+    url = "http://images.cocodataset.org/val2017/000000039769.jpg"
```
Comment (nvukobratTT) [inline model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py:70]: Cna you keep original image?

File: model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py [hunk 2]
```diff
@@ -65,11 +67,14 @@ def run_pytorch_yolov5_320(variant="yolov5s"):
...
+    os.system(f"wget -c -P /tmp/ --read-timeout=5 --tries=0 {shlex.quote(url)}")
```
Comment (nvukobratTT) [inline model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py:71]: Can wget return the downloaded path? Or we need to manually specify it?

File: model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py (hunk 1 above)
Comment (pdevi5) [reply model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py:70]: sure will keep original image

File: model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py (hunk 2 above)
Comment (pdevi5) [reply model_demos/cv_demos/yolo_v5/pytorch_yolov5_320.py:71]: Sure will update on upcoming patch

**Review by nvukobratTT — APPROVED** [review]

**Pull request closed without merging**